tests/data/simple.cgx

```diff
@@ -0,0 +1,10 @@
+<template>
+	<label text="Simple"/>
+</template>
+
+<code>
+import collagraph as cg
+
+class Simple(cg.Component):
+		pass
```

Comment:
TABS?! :)

Reply:
ai! 😅 gotta fix my editor

Reply:
Good to see that it still worked though ;) If people decide to use 2 spaces as indent for `.cgx` files, that would still be ok apparently.